cmd/test.go

```diff
@@ -173,23 +211,24 @@ func Run(out *output.OutWriter) bool {
 			Tag:          tag,
 			OutputStream: os.Stdout,
 		}, docker.AuthConfiguration{}); err != nil {
-			logrus.Fatalf("Error pulling remote image %s: %s", imagePath, err.Error())
+			ctc_lib.Log.Fatalf("Error pulling remote image %s: %s", imagePath, err.Error())
 		}
 	}
 
 	if driver == drivers.Host && !utils.UserConfirmation(warnMessage, force) {
-		return false
+		// Force Success. Skipping Running tests as directed by the user.
+		return
```

Comment:
I think we might want to force failure here? Similar to how ctrl+c usually causes applications to exit 1.

Reply:
umm, i am of opinion that if user pressed 'n'  it not an tool error as the tool did what the user asked him to do.

Reply:
I changed my mind and error out "User Aborted" Message